cli/command/deploy.go

```diff
@@ -1,15 +1,169 @@
 package command
 
-import "github.com/urfave/cli"
+import (
+	"fmt"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+	"io/ioutil"
+	"strconv"
+)
 
 type DeployCommand struct {
 	*CommandBase
 }
 
 func NewDeployCommand(b *CommandBase) *DeployCommand {
-	return &DeployCommand{b}
+	return &DeployCommand{
+		CommandBase: b,
+	}
 }
 
 func (d *DeployCommand) Command() cli.Command {
-	return cli.Command{}
+	return cli.Command{
+		Name:  "deploy",
+		Usage: "manage layer0 deploys",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new deploy",
+				Action:    d.create,
+				ArgsUsage: "PATH NAME",
```

Comment:
[nit] Based on the discussion about the ArgsUsage `NAME` becoming `DEPLOY_NAME`. I think this is the currently agreed consensus on the discussions I saw about this.